patch: Update serv-vars.md (#2026)
balena-io/docs: pages/learn/manage/serv-vars.md

> Environment and service variables defined at the fleet level are available to all devices in that fleet unless they are redefined with a variable of the same name at the device level.

Reviewer: ```suggestion
Variables defined at the fleet level are available to all devices in that fleet unless they are redefined with a variable of the same name at the device level.
```

> They can be found from the fleet summary page under the *Environment Variables* and *Service Variables* tabs.

Reviewer: ```suggestion
They can be found from the fleet summary page under the *Variables* tab.
```

The {{ $names.company.lower }} CLI and SDKs all include methods to easily read, add or update environment and service variables. Consult the appropriate reference for code examples.

* [CLI environment reference](/tools/cli/#envs)

> * [Node.js SDK environment reference](/reference/sdk/node-sdk/#balena.models.fleet.envVar)

Reviewer: ```suggestion
* [Node.js SDK environment reference](/reference/sdk/node-sdk/#balena.models.application.envVar)
```
